### PR TITLE
Feature/display books

### PR DESCRIPTION
**Pull request by shaqdeff**
- Changed the default state in the books reducer from an empty array to an array with a few books.
- Imported Redux Provider and store in the main component of the React app.
- Connected the app to the Redux store.
- Used useSelector to consume the state.
- Import Redux useDispatch to add and remove books, and check status.

Followed all other project requirements.

Kindly review and provide feedback.
Regards.

### CODE REVIEW

File: src/components/AddBook.jsx [hunk 1]
```diff
@@ -1,37 +1,50 @@
...
+        <input
+          type="text"
+          id="author"
+          name="author"
+          value={state.author}
+          placeholder="Author"
+          onChange={handleChange}
+        />
         <button type="submit">Add Book</button>
       </form>
```
Comment (zieeco) [inline src/components/AddBook.jsx]: - Please, kidly ensure the form doesn't submit when the input fields are empty. Right now when the user clicks the add button without filling any inputs, the form is submitted. please fix this issue.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @shaqdeff ,


Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/AddBook.jsx (hunk 1 above)
Comment (shaqdeff) [reply src/components/AddBook.jsx]: Hey @zieeco, I seem to be stuck on this particular issue. I have used preventDefault(), but the form is still submitting. I have no idea what's up. Mind helping?

Comment (shaqdeff) [reply src/components/AddBook.jsx]: Rectified the issue. Thank you.

**Review by elisha2kyakpo1 — APPROVED** [review]
Hi 👋 @shaqdeff, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- You have done a great job working on the requested changes ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**